go/oasis-node/cmd/registry/entity/entity.go

```diff
@@ -61,15 +64,17 @@ var (
 	}
 
 	registerCmd = &cobra.Command{
-		Use:   cmdRegister,
-		Short: "register an entity",
-		Run:   doRegisterOrDeregister,
+		Use:     cmdRegister,
+		Aliases: []string{"register"},
+		Short:   "generate a register entity transaction",
+		Run:     doRegisterOrDeregister,
 	}
 
 	deregisterCmd = &cobra.Command{
-		Use:   "deregister",
-		Short: "deregister an entity",
-		Run:   doRegisterOrDeregister,
+		Use:     "gen_deregister",
+		Aliases: []string{"deregister"},
```

Comment:
Why aliases?

Reply:
I actually realize the aliasing was not a great idea because it's a breaking change to the CLI anyhow. So I can remove. The original idea was that I wanted to keep the sub-commands on the CLI consistent so people wouldn't be confused but this actually is more confusing.